src/redux/reducer.ts

```diff
@@ -352,16 +352,23 @@ export type StudyUpdated = {
     force: number; //IntRange<0, 1>;
 } & (StudyUpdatedUndefined | StudyUpdatedStudy);
 
-export type TreeNodeData = {
-    parentNodeUuid: UUID;
+type NodeCommonData = {
     label: string;
-    description: string | null;
+    parentNodeUuid?: UUID;
     globalBuildStatus?: BUILD_STATUS;
-    localBuildStatus?: BUILD_STATUS;
+    description?: string;
     readOnly?: boolean;
-    caseName?: string;
 };
-export type CurrentTreeNode = Node<TreeNodeData> & { id: UUID };
+export type ReactFlowModificationNodeDataType = NodeCommonData & { localBuildStatus?: BUILD_STATUS };
+
+export type ModificationNode = Node<ReactFlowModificationNodeDataType, 'NETWORK_MODIFICATION'> & {
```

Comment:
Remove `Type` at thje end of `ReactFlowModificationNodeDataType`

Reply:
done